src_erl/NerlMonitor/README.md

```diff
@@ -0,0 +1,41 @@
+NerlMonitor
+=====
+NerlMonitor is an external tool that helps NErlNet users get more knowledge on the experiment flow and also gain the ability to terminate workers mid-experiment. This can help in monitoring your model behavior for different kinds of failures. The app is also used to gain various statistics.
+
+
+# Dependencies
+`pip install` these libraries (In addition to the src_py/requirements.txt file):
+- NetowrkX
```

Comment:
@GuyPerets106 
Create a requirements.txt file
You can use pipreqs 
https://stackoverflow.com/questions/31684375/automatically-create-requirements-txt

Change instructions to:
1. Create a python virtualenv for NerlMonitor. 
2. Install modules: pip3 install -r /usr/local/lib/nerlnet-lib/NErlNet/src_py/NerlMonitor/requirements.txt